dev.js

```diff
@@ -13,7 +13,7 @@ ngrok
       opn(url);
       console.log(`🌍 Available online: ${url}`);
     }
-    console.log(`👩🏻‍💻  Webhook URL for Stripe: ${url}/webhook`);
+    console.log(`👩🏻‍💻  Webhook URL for Stripe: ${url}/webhooks`);
```

Comment:
Nit: technically it's just one webhook which handles multiple events, no?

Reply:
I've seen it written both ways and confuse myself every time. I'm happy with either really :)